.pre-commit-config.yaml

```diff
@@ -19,13 +19,13 @@ repos:
       - id: isort
         args: ["--profile", "black"]
   - repo: https://github.com/PyCQA/docformatter
-    rev: master
+    rev: v1.7.5
```

Comment:
I think this need to be reverted, right @alecandido ?

Reply:
Yes and no: that will have to be done at each and every iteration. We may do it, and postpone the problem once more, but we may even invest in a longer term solution...

I would abandon Pylint, Black and docformatter, in favor of Ruff, which is doing everything (including [formatting docstrings](https://docs.astral.sh/ruff/formatter/#docstring-formatting)).

We just need something like:
```yaml
  - repo: https://github.com/astral-sh/ruff-pre-commit
    rev: v0.8.2
    hooks:
      # Run the linter.
      - id: ruff
        args: [ --fix ]
      # Sort imports
      - id: ruff
        args: [ --select, I, --fix ]
      # Run the formatter.
      - id: ruff-format
```
and 
```toml
[tool.ruff.format]
docstring-code-format = true
```

It won't format the plain text, but we can bare with it :)